src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -27,31 +29,35 @@
     /**
      * Parses the given {@code String} of arguments in the context of the AddCommand
      * and returns an AddCommand object for execution.
-     * @throws ParseException if the user input does not conform the expected format
+     *
+     * @throws ParseException
+     *             if the user input does not conform the expected format
```

Comment:
/nit Similar to above. I think we should decide whether to keep @param on the same line or two different lines. Might need to check the coding style standard for cs2103.